library/src/main/java/com/novoda/downloadmanager/lib/DownloadManagerJobCreator.java

```diff
@@ -0,0 +1,15 @@
+package com.novoda.downloadmanager.lib;
+
+import com.evernote.android.job.Job;
+import com.evernote.android.job.JobCreator;
+
+class DownloadManagerJobCreator implements JobCreator {
+    @Override
+    public Job create(String tag) {
```

Comment:
I though you were referring to the tag.

That method is called by the jobScheduler library through the library instantiation `JobManager.create(context).addJobCreator(jobCreator);`